packages/rum-core/src/common/service-factory.js

```diff
@@ -52,20 +49,12 @@ class ServiceFactory {
         serviceFactory.getService('LoggingService')
       )
     })
-
-    this.registerServiceInstance('PatchUtils', patchUtils)
-    this.registerServiceInstance('Utils', utils)
   }
-  init() {
-    if (this.initialized) {
```

Comment:
This doesn't seem to be related to the rest of the changes - is it necessary? This change seems like something that could have subtle ramifications, so I'd prefer if it could be done separately if possible.

Reply:
Sure will separate it out. Thanks for the review @axw